pkg/fqdn/cache.go

```diff
@@ -760,7 +760,11 @@ type DNSZombieMapping struct {
 	IP netip.Addr `json:"ip,omitempty"`
 
 	// AliveAt is the last time this IP was marked alive via
```

Comment:
Incidentally now I read the comment above `DNZombieMapping`, it refers to `zero AliveAt` so that comment also needs updating.